app/docs/form-engine-client-setup/sections/quickbase.section.html

```diff
@@ -0,0 +1,27 @@
+<doc-section>
+  <doc-header id="quickbase">Quickbase</doc-header>
+  <doc-caption>Actions required to be performed on quickbase</doc-caption>
+
+  <h5>Copy Application</h5>
```

Comment:
Specify which application. Check with Henry regarding which instance (dev or prod) he would like us to copy. It shouldn't really matter but we should be consistent.